src/widgets/buttons.ts

```diff
@@ -478,6 +488,42 @@ export function deleteButtonWithCheck (
   return deleteButtonElt
 }
 
+function getButtonStyle (options: ButtonWidgetOptions = { buttonType: ButtonType.Primary, filled: true }) {
+  // We need to accomadate for legacy code, which is why we have to allow buttonType and filled to be optional
+  let backgroundColor: (ButtonType | string | undefined) = options.buttonType
+  let fontColor: (ButtonType | string | undefined) = '#ffffff'
+  let borderColor: (ButtonType | string | undefined) = options.buttonType
+  let hoverBackgroundColor: (ButtonType | string | undefined) = `lighten(${options.buttonType}, 5%)`
+  let hoverFontColor: (ButtonType | string | undefined) = fontColor
+  if (!options.filled) {
+    backgroundColor = '#ffffff'
+    fontColor = options.buttonType
+    borderColor = options.buttonType
+    hoverBackgroundColor = options.buttonType
+    hoverFontColor = backgroundColor
+  }
+
+  // See https://design.inrupt.com/atomic-core/?cat=Atoms#Buttons
+  return {
+    'background-color': `${backgroundColor}`,
+    color: `${fontColor}`,
+    'font-family': 'Raleway, Roboto, sans-serif',
```

Comment:
This font is not showing up because we need to add the font family to the library itself.  I will create a ticket.  should work once this is done.

Reply:
See #293.